src/main/java/org/casbin/jcasbin/main/ManagementEnforcer.java

```diff
@@ -15,8 +15,8 @@
 package org.casbin.jcasbin.main;
 
 import com.googlecode.aviator.runtime.type.AviatorFunction;
+import com.sun.deploy.util.StringUtils;
```

Comment:
This import is not necessary. jCasbin is based on JDK 1.8, which has built-in string join support. In fact, jCasbin already uses it:

https://github.com/casbin/jcasbin/blob/e5d1d27bd781e8f697f83a402569320516735ce3/src/main/java/org/casbin/jcasbin/util/Util.java#L180-L188

Reply:
Fixed